sips/sip-172.md

```diff
@@ -0,0 +1,71 @@
+---
+sip: 172
+title: Governance V2
+status: Draft
+author: Andy T CF (@andytcf), CryptoToit (@CryptoToit)
+discussions-to: TBD
+
+created: 2021-07-30
+---
+
+<!--You can leave these HTML comments in your merged SIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new SIPs. Note that an SIP number will be assigned by an editor. When opening a pull request to submit your SIP, please use an abbreviated title in the filename, `sip-draft_title_abbrev.md`. The title should be 44 characters or less.-->
+
+# Simple Summary
+
+This proposal seeks to elevate the current Synthetix Governance System, addressing pain points that have been identified during the first version of governance as well as proposing new processes and tools to improve the self-autonomy and efficiency of the Synthetix DAOs.
```

Comment:
@andytcf @CryptoToit see below comment. Feel we should describe the problems in more detail in the SIP